index.html

```diff
@@ -71,6 +71,51 @@
         This document is a work in progress.
       </p>
     </section>
+    <section class="informative">
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        The Screen Orientation API allows web applications that would be
+        negatively impacted by users changing the orientation of their device
+        to block any changes. It also provides the current screen orientation
+        and any orientation changes so all web applications can display
+        correctly.
+      </p>
+      <p>
+        This specification describes how the API:
+      </p>
+      <ul>
+        <li>provides the current screen orientation type and angle of a device
+        as defined in the <a>updating orientation algorithm</a>.
+        </li>
+        <li>updates the screen orientation when the viewport angle changes
+        according to the <a>screen orientation change algorithm</a>.
+        </li>
+        <li>locks the screen to a specific orientation following the <a>locking
+        orientation algorithm</a>.
+        </li>
+        <li>unlocks the locked screen to the default orientation as described
+        in the <a data-link-for="ScreenOrientation">unlock()</a> method.
```

Comment:
Don't we have an unlock algorithm? Or are the steps to unlock explicitly part of the .unlock()?

Reply:
No there is no unlock algorithm.  The unlock() method basically says to run the steps to lock and set to default orientation.

Reply:
I wonder if we should abstract out the unlocking steps from the method? Do any algorithms  unlock by calling into .unlock()?

Reply:
I agree it probably makes sense to have unlock() as an algorithm.  I'll leave it as it is for now, and when I have written it, I will change the wording/link

Reply:
yeah, we can do it as a new PR.